test/util.go

```diff
@@ -66,6 +67,7 @@ func CreateClient() client.Client {
 		cli, err = client.NewClient(configUrl, fullUserName, passWord, sysUserName, sysPassWord, config.NewDefaultClientConfig())
 	}
 	if err != nil {
+		log.Warn("BOOT", nil, "fail to create client")
```

Comment:
test utils no need to print log, error message is enough

Reply:
down